platforms/python/prereqs/build.sh

```diff
@@ -10,14 +10,11 @@ wget https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_
 wget https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc -O /python.tar.xz.asc
 
 debianFlavor=$DEBIAN_FLAVOR
+debianHackFlavor=$DEBIAN_HACK_FLAVOR
 pythonSdkFileName=""
+PYTHON_GET_PIP_URL="https://github.com/pypa/get-pip/raw/3cb8888cc2869620f57d5d2da64da38f516078c7/public/get-pip.py"
```

Comment:
Is this URL (and the SHA variable down below) tied to a specific version of pip? Do these two variables need to be updated whenever we updated the version of pip used in the `__pythonVersions.sh` script?